php/WP_CLI/Runner.php

```diff
@@ -570,7 +570,7 @@ private function generate_ssh_command( $bits, $wp_command ) {
 		}
 
 		$is_tty             = function_exists( 'posix_isatty' ) && posix_isatty( STDOUT );
-		$docker_compose_cmd = ! empty( Process::create( Utils\esc_cmd( 'docker compose %s', 'version' ) )->run()->stdout )
+		$docker_compose_cmd = ! empty( Process::create( Utils\esc_cmd( '/usr/bin/env docker compose %s', 'version' ) )->run()->stdout )
```

Comment:
Should we use `Utils\force_env_on_nix_systems()` here? That's what db-command uses:

https://github.com/wp-cli/db-command/blob/41b502bdd8121134d96baf0e3cd681c0c039f87f/src/DB_Command.php#L611

Reply:
Makes sense for the sake of Windows compatibility.